gcam-usa.md

```diff
@@ -225,8 +225,13 @@ This approach differs somewhat from the global GCAM, where future cooling shares
 
 
 Non-CO2 GHGs
-============
-Note that at this time the specification of Non-CO2 greenhouse gases in GCAM-USA is incomplete.  This is an active area of development.  However for this reason the climate model will be disabled in GCAM-USA scenarios.
+===========
+GCAM-USA represents state-level CH4 and N2O emissions in electricity, buildings, industrial energy use, industry processes, urban processes, refining, and N Fetilizer sectors. HFCs are represented in building sector (for residential and commercial cooling). SF6 are use in electricity transmission system (electricity_net_ownuse at grid level). Emissions from resource production and agricutral sectors have not been downscaled to states yet.
+
+Historical CH4 and N2O emissions in industry processes and urban processes are calibrated to the [2022 U.S. State-level Non-CO2 GHG Mitigation Report](gcam-usa.html#epa2022). Emissions in other sectors used the same emission factors from region "USA" in the global 32-region model, developed from the [2019 EPA Global Non-CO2 Greenhouse Gas Emission Projection & Mitigation Potential Report](emissions.html#epa2019).
+
+In the current version, marginal abatement cost(MAC) curves are assumed to be the same as region "USA" in the global 32-region model. In future development, we will update state-level MACs for all sectors based on the [2022 U.S. State-level Non-CO2 GHG Mitigation Report](gcam-usa.html#epa2022).

```

Comment:
Please add a space between `cost` and `(MAC)`.

Reply:
I suggest replacing `the same as region "USA" in the global 32-region model` with `the same as the "USA" region in the 32-region GCAM model`.